apps/zotonic_mod_base/src/filters/filter_is_a.erl

```diff
@@ -25,9 +25,7 @@ is_a({rsc_list, List}, Cat, Context) ->
 is_a(Arg, Cat, Context) ->
     case m_category:name_to_id(Cat, Context) of
         {ok, CatId} -> z_list_of_ids_filter:filter(Arg, fun(Id) -> m_rsc:is_a(Id, CatId, Context) end, Context);
-        {error, {unknown_category, _}} -> false;
-        {error, _Reason} when is_integer(Arg) -> false;
```

Comment:
I think this `when` is too strict.

The check on is_list might move up and this one might just be the default.

Applying the `is_a` filter on a list (or list-alike thing) should perform like a filter on some category.  That is why the when on `is_list(Arg)` is there, to return an empty list.

Reply:
Ah... the error clauses match on the input, not on the error itself.

Reply:
Going to do this later today. I'll refactor it a bit to clarify a bit more that `is_a` produces different answers when there is no match depending on the input.

Reply:
The `{rsc_list, _}` is now also a record: `#rsc_list{ list = List }`.